nimble/host/mesh/src/proxy.c

```diff
@@ -877,7 +878,7 @@ bool bt_mesh_proxy_relay(struct os_mbuf *buf, u16_t dst)
 		struct bt_mesh_proxy_client *client = &clients[i];
 		struct os_mbuf *msg;
 
-		if (!client->conn_handle) {
+		if (clients[i].conn_handle == BLE_HS_CONN_HANDLE_NONE) {
```

Comment:
since there is `*client`, let us use it:
 if (client->conn_handle == BLE_HS_CONN_HANDLE_NONE) {}